app/javascript/components/Results/Results.tsx

```diff
@@ -128,15 +132,27 @@ export const Results = (props: ResultsProps) => {
                 </Grid>
               </GridContainer>
             );
-          })}
+          })) : (
+            <GridContainer className='result search-result-item'>
+              <Grid row>
+                <Grid tablet={{ col: true }}>
+                  <h4>Sorry, no results found for &#39;{props.query}&#39;. Try entering fewer or more general search terms.</h4>
```

Comment:
This is temporary hard-coded language that will be removed once we introduce internationalization.